packages/frontend/web/components/AdSlot.tsx

```diff
@@ -71,3 +85,24 @@ export const AdSlot: React.FC<{
         />
     );
 };
+
+export const AdSlot: React.FC<{
+    asps: AdSlotParameters;
+    config: ConfigType;
+}> = ({ asps, config }) => {
+    if (!shouldDisplayAdvertisements(config)) {
+        return null;
+    }
+    return (
+        <AdSlotCore
+            name={asps.name}
```

Comment:
If you're passing everything through, and they're all the same names, you can probably do `{...asps}` here.

Reply:
Thank you so much. I knew there was a notation for that, but I wasn't sure what it was.

Reply:
Refactoring pushed